data_processing/month_processing.py

```diff
@@ -0,0 +1,187 @@
+# ---
```

Comment:
This file doesn't have the ML stuff in it, was using it to cross validate data before hand